src/block/columns/edit.js

```diff
@@ -146,6 +149,17 @@ const Edit = props => {
 	)
 }
 
+addFilter( 'stackable.columns.edit.blockClassNames', 'stackable-columns-has-single-block-polyfill', classes => {
+	const { numInnerBlocks } = useBlockContext()
+	const userAgent = navigator?.userAgent
+
+	if ( userAgent?.indexOf( 'Firefox' ) !== -1 && numInnerBlocks === 1 ) {
+		classes.push( 'stk-block-columns--has-single-block-polyfill' )
+	}
+
+	return classes
+} )
+
```

Comment:
I think this might work, if we do this, we optimize the `addFilter` so it only gets added for Firefox. wdyt?
```suggestion
const userAgent = navigator?.userAgent || ''
if ( userAgent.indexOf( 'Firefox' ) !== -1 ) {
	addFilter( 'stackable.columns.edit.blockClassNames', 'stackable-columns-has-single-block-polyfill', classes => {
		const { numInnerBlocks } = useBlockContext()

		if ( numInnerBlocks === 1 ) {
			classes.push( 'stk-block-columns--has-single-block-polyfill' )
		}

		return classes
	} )
}
```